ChatApp/Custom+UI/ChatView.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  ChatView.swift
+//  ChatApp
+//
+//  Created by bakhva  on 29.04.23.
+//
+
+import UIKit
+
+final class ChatView: UIView {
+    
+    // MARK: Text Field View
+    let TextFieldContainer: TextFieldContainerView = {
+        let container = TextFieldContainerView()
+        container.translatesAutoresizingMaskIntoConstraints = false
+        container.textField.font = .systemFont(ofSize: 14)
+
+        return container
+    }()
+    
+    // MARK: Chat Table View
+    private let chatTableView: UITableView = {
+        let tableView = UITableView()
+        tableView.backgroundColor = .clear
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return tableView
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configUI()
+        addSubviews()
+        addConstraints()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
```

Comment:
The code will be much nicer if you write this private view builder functions in a private extension